tools/upgrademonitor/go.mod

```diff
@@ -0,0 +1,13 @@
+module github.com/celestiaorg/celestia-app/tools/upgrademonitor
```

Comment:
What was the rationale for this having it's own go.mod?

Reply:
hmm upgrademonitor is distinct from celestia-app in that the binary will be used independently of celestia-app. A distinct go.mod allows upgrademonitor to manage its own dependencies.

Your question made me realize that we could even extract upgrademonitor entirely to its own repo.

Reply:
If we want it in a stand alone repo: https://github.com/celestiaorg/upgrade-monitor/pull/1